nautobot/extras/templates/extras/computedfield.html

```diff
@@ -93,4 +89,16 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
     </div>
 </div>
 
+<div class="row">
+	<div class="col-md-12">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <strong>Template</strong>
+            </div>
+            <span><pre style="margin: 15px;">{{ object.template }}</pre></span>
```

Comment:
What's the custom `margin` styling needed for?

Also, the `<span>` probably isn't needed.

Reply:
The margin adds a bit of distance between the itself and the panel.